tables/isolate_mutations.csv

```diff
@@ -13238,3 +13238,112 @@ hCoV-19/Germany/BY-MVP-000010417/2021,N,32,del,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,RdRP,323,L,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,nsp14,42,V,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,nsp15,122,F,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,nsp15,122,I,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,nsp15,122,L,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,nsp15,122,V,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,67,V,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,69,del,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,70,del,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,95,I,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,142,D,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,143,del,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,144,del,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,145,del,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,211,I,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,212,del,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,214,ins,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,339,D,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,371,L,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,373,P,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,375,F,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,417,N,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,440,K,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,446,S,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,477,N,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,478,K,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,484,A,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,493,R,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,496,S,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,498,R,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,501,Y,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,505,H,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,547,K,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,614,G,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,655,Y,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,679,K,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,681,H,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,764,K,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,796,Y,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,856,K,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,954,H,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,969,K,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,S,981,F,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,E,9,I,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,M,3,G,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,M,19,E,NULL,NULL
+hCoV-19/Botswana/R40B58_BHP_3321001245/2021,M,63,T,NULL,NULL

```

Comment:
Please ignore the mutations not in S (Spike) protein. Because this paper is using pseudovirus assay.